### DISCUSSION TITLE
Preferred way to customize nginx.conf

### DISCUSSION THREAD

**Discussion by front-stream** [serversideup/docker-php#459] [Q&A]
What is the preferred way to customize nginx.conf file. For example. I would like to add proxy cache. I think that documentation should contain info how to do that.

**Comment by jaydrogers** [comment]
Good point!

Try adding your extra configs in this directory. See how we include it?

https://github.com/serversideup/docker-php/blob/143597be006fc7872561b7509f0249c00f28c74a/src/variations/fpm-nginx/etc/nginx/site-opts.d/http.conf.template#L45-L46
